internal/cmd/rem.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/keygen-sh/keygen-relay/internal/licenses"
+	"github.com/spf13/cobra"
+)
+
+func RemCmd(manager licenses.Manager) *cobra.Command {
+	var id string
+
+	cmd := &cobra.Command{
+		Use:   "rem",
```

Comment:
After a little thought — for consistency, let's rename `rem` to `del` to match [our main CLI](https://github.com/keygen-sh/keygen-cli?tab=readme-ov-file#delete-a-release).

I've updated the spec to reflect this.

Reply:
Done!